build.cake

```diff
@@ -438,11 +446,10 @@ void CopyMonoBuild(BuildEnvironment env, string sourceFolder, string outputFolde
     FileHelper.Delete(CombinePaths(outputFolder, "System.Threading.Thread.dll"));
 }
 
-string PublishMonoBuild(BuildEnvironment env, BuildPlan plan, string configuration, bool archive)
+string PublishMonoBuild(string project, BuildEnvironment env, BuildPlan plan, string configuration, bool archive)
 {
     Information("Publishing Mono build...");
```

Comment:
It might be worth printing the project name that's being published so that the text isn't just duplicated in the build log.